cmd/slack-agent-client/cmd/root.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/cybozu-go/github-actions-controller/agent"
+	"github.com/spf13/cobra"
+)
+
+var clientConfig struct {
+	server      string
+	namespace   string
+	jobInfoFile string
+	result      string
+	channel     string
+	extend      bool
+	dryRun      bool
+}
+
+var rootCmd = &cobra.Command{
+	Use:   "slack-agent-client PODNAME [RESULT]",
+	Short: "slack-agent-client sends job result to Slack agent",
+	Long:  "slack-agent-client sends job result to Slack agent",
```

Comment:
I think RESULT should be explained.
It will accept `success`, `failure`, `canceled` and `unknown`.

Reply:
I added explanation.